mindsdb/libs/data_sources/maria_ds.py

```diff
@@ -0,0 +1,5 @@
+from mindsdb.libs.data_sources.mysql_ds import MySqlDS
+
+
+class MariaDS(MySqlDS):
```

Comment:
Copy-paste the code from the MysqlDS, since it's short *and* we might make divergent changes to these 2 later.

I know it's boilerplate but I plan on removing the boilerplate later from all database sources by having shared function between them, when we add more functionality, so that's fine.